### PR TITLE
94 packaging

### PR DESCRIPTION
**Pull request by jperedadnr**

### CODE REVIEW

File: src/main/java/com/gluonhq/NativeBaseMojo.java [hunk 1]
```diff
@@ -144,6 +153,11 @@ private void configSubstrate() {
                 break;
             case Constants.PROFILE_IOS:
                 targetTriplet = new Triplet(Constants.Profile.IOS);
+                IosSigningConfiguration signingConfiguration = new IosSigningConfiguration();
```
Comment (johanvos) [inline src/main/java/com/gluonhq/NativeBaseMojo.java]: Isn't it better to have this in Substrate? We now leak the IosSigningConfiguration to the plugin, which makes it much harder to modify that class. If we only send the Strings (identity/profile) to Substrate, this code block can be executed there.

Comment (jperedadnr) [reply src/main/java/com/gluonhq/NativeBaseMojo.java]: How would you send them? The only way will be adding these fields in the public `ProjectConfiguration`, which already has an exposed `IosSigningConfiguration` object.

Comment (johanvos) [reply src/main/java/com/gluonhq/NativeBaseMojo.java]: As Strings.

Comment (jperedadnr) [reply src/main/java/com/gluonhq/NativeBaseMojo.java]: I've created a follow-up issue for this

**Pull request merged**